torch/_torch_docs.py

```diff
@@ -7754,3 +7754,126 @@ def merge_dicts(*dicts):
     >>> g_cpu.device
     device(type='cpu')
 """)
+
+add_docstr(torch.searchsorted,
+           r"""
+searchsorted(sorted_sequence, values, out_int32=False, right=False, out=None) -> Tensor
+
+Find the indices from the *innermost* dimension of :attr:`sorted_sequence` such that, if the 
```

Comment:
Funnily enough, here I like the sorting terminology better than the bin terminology. I guess it depends on the context (what the function is named)